typings/index.ts

```diff
@@ -19,14 +19,11 @@ export interface Mask {
  *  @param {Mask} mask - Which mask to apply
  */
 export interface RedacItConfig {
-  fields: string[];
+  fields: (string | RegExp)[];
   mask?: Mask;
 }
 
-export type ReplacerFunction = (
-  key: string,
-  value: string
-) => string | undefined;
+export type ReplacerFunction = (key: any, value: any) => any;
```

Comment:
I am not sure about the typings here. Why changing them all to `any`?

Reply:
This is to respect the official typings for `JSON.stringify`, which use `any`.

keys and values may not be strings!